.github/workflows/linux.yml

```diff
@@ -0,0 +1,43 @@
+name: linux
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+  schedule:
+    - cron: "0 5 * * 1"  # runs at 05:00 UTC on Mondays
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      os: Linux-x86_64
+    strategy:
+      fail-fast: true
+      max-parallel: 4
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+      - name: Install
+        shell: bash -l {0}
+        run: |
+          conda config --set always_yes yes --set changeps1 no
+          conda config --add channels conda-forge
+          conda install conda-devenv
+          conda update -q conda
+          conda info -a
+          conda devenv
+          source activate optima
+          cmake -P install
```

Comment:
I prefer to put everything inside the workflow script like that, this way a new user can easily come here and check the steps to build and test. But this is not a strong preference.

Reply:
This is fine for now. It's the same code for Linux and macOS though.